docker/.env

```diff
@@ -0,0 +1,6 @@
+CASSANDRA_IMAGE=bazaarvoice/cassandra
```

Comment:
any objections to merging CASSANDRA_IMAGE and CASSANDRA_TAG into one variable?

Reply:
for whole consistency
because image and tag are different vars